seleniumByTechnocredits/src/seleniumPractise/saket/Assignment6.java

```diff
@@ -0,0 +1,207 @@
+
+			element.click();
+			String sixth = alert.getText();
+			alert.dismiss();
+
+			if (sixth.equals("Please enter your current organization")) {
+
+				System.out.println("sixth alert is validated");
+			}
+
+			driver.findElement(By.id("organization")).sendKeys("capgemini");
+			JavascriptExecutor js7 = (JavascriptExecutor) driver;
+			js.executeScript("arguments[0].scrollIntoView(true)", element);
+
+			element.click();
+			String seventh = alert.getText();
+			alert.dismiss();
+			if (seventh.equals("Username is mandatory field.")) {
+
+				System.out.println("Seventh alert is validated");
+			}
+
+			driver.findElement(By.xpath("//input[@id='usernameReg']")).sendKeys("234");
+			element.click();
+
+			String eight = alert.getText();
+			alert.dismiss();
+
+			if (eight.equals("Username length should be greater then 5 characters.")) {
+
+				System.out.println("eight alert validated");
+
+			}
+
+			driver.findElement(By.xpath("//input[@id='usernameReg']")).sendKeys("2343r3435345");
+			element.click();
+
+			String nineth = alert.getText();
+			alert.dismiss();
+
+			if (nineth.equals("password is mandatory field.")) {
+
+				System.out.println("nineth alert validated");
+
+			}
+
+			driver.findElement(By.id("passwordReg")).sendKeys("123");
+			element.click();
+
+			String tenth = alert.getText();
+			alert.dismiss();
+
+			if (tenth.equals("password length should be greater then 5 characters.")) {
+
+				System.out.println("tenth alert validated");
+
+			}
+			driver.findElement(By.id("passwordReg")).clear();
+			driver.findElement(By.id("passwordReg")).sendKeys("saket123");
+			System.out.println(driver.findElement(By.id("passwordReg")).getAttribute("value"));
+
+			element.click();
+
+			String eleventh = alert.getText();
+			alert.dismiss();
+			if (eleventh.equals("please reenter password")) {
+
+				System.out.println("eleventh alert validated");
+
+			}
+
+			driver.findElement(By.id("repasswordReg")).sendKeys("12");
+			element.click();
+			String twelve = alert.getText();
+			alert.dismiss();
+			if (twelve.equals("retype password donot match.")) {
+
+				System.out.println("twelth alert validated");
+
+			}
+
+			driver.findElement(By.id("repasswordReg")).clear();
+			Thread.sleep(1000);
+			driver.findElement(By.id("repasswordReg")).sendKeys("saket123");
+			element.click();
+
+			//System.out.println(driver.findElement(By.id("repasswordReg")).getAttribute("value"));
+
+			String thirteen = alert.getText();
+			System.out.println(thirteen);
+			alert.dismiss();
+			if (thirteen.equals("Please agree to terms of service and privacy policy")) {
+
+				System.out.println("thirteen alert validated");
+
+			}
+
+			driver.findElement(By.xpath("//input[@value='agree this condition']")).click();
+			element.click();
+
+			String fourteen = alert.getText();
+			alert.dismiss();
+			if (fourteen.equals("Success")) {
+
+				System.out.println("fourteenthFf alert validated");
+
+			}
+
+		}
+	}
+
+}
```

Comment:
Code looks good to me Saket.Just a suggestion if you look at the code you can optimise the code and reduce line sof code
Please try to write using methods instead of writing everything in main.Split using methods.

Reply:
sure will do it in future code